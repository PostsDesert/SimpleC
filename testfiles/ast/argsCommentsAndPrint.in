int basefunc(int d, int g) {
    // Go D Guild!
    print(d);
    print();
}