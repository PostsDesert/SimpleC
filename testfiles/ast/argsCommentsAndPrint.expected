Program
  List
    FuncDesc
      IdDecl ID="basefunc"
      List
        IdDecl ID="d"
        IdDecl ID="g"
      Block
        List
          FuncCallStmt
            FuncCall
              IdUse ID="print"
              List
                IdUse ID="d"
          FuncCallStmt
            FuncCall
              IdUse ID="print"
              List
